components/apimgt/org.wso2.carbon.apimgt.rest.api/src/main/java/org/wso2/carbon/apimgt/rest/api/impl/ApisApiServiceImpl.java

```diff
@@ -49,14 +49,28 @@
     public Response apisGet(String limit,String offset,String query,String type,String sort,String accept,String ifNoneMatch){
         List<org.wso2.carbon.apimgt.api.model.API> apis;
         List<APIDTO> list = new ArrayList<APIDTO>();
+
+        boolean isTenantFlowStarted = false;
+
         try {
-            provider = APIManagerFactory.getInstance().getAPIProvider("admin");
-            apis = provider.searchAPIs(query,type, "admin");
+            String loggedInUser = CarbonContext.getThreadLocalCarbonContext().getUsername();
+            APIProvider apiProvider = RestApiUtil.getProvider(loggedInUser);
+            String tenantDomain =  CarbonContext.getThreadLocalCarbonContext().getTenantDomain();
+            if(tenantDomain != null && !MultitenantConstants.SUPER_TENANT_DOMAIN_NAME.equals(tenantDomain)) {
+                isTenantFlowStarted = true;
```

Comment:
The tenants flows have been moved currently from impl level to CXF handler level
Fixed comments in #1091 commit : 897ef52